src/main/java/com/checkmarx/flow/service/FlowService.java

```diff
@@ -3,6 +3,7 @@
 import com.checkmarx.flow.dto.ScanRequest;
 import com.checkmarx.flow.exception.MachinaRuntimeException;
 import com.checkmarx.sdk.dto.ScanResults;
+import jline.internal.Log;
```

Comment:
I'm not sure we need this import since this class already uses the '@Slf4j' annotation

Reply:
removed